include/inicpp.h

```diff
@@ -207,12 +208,29 @@ namespace ini
                 str = "";
         }
 
+        void erase_comment(std::string &str)
```

Comment:
Please change the function name to Camel-Case to match the variable naming style.

Reply:
Sure. Fixed in the following commit.